tests/ut_seattlelibv2_repypp_multiplearguments.py

```diff
@@ -0,0 +1,45 @@
+#pragma out Invalid number of arguments
+#pragma out repypp.py infile outfile
+
+#pragma out preprocesses infile and includes content from the current directory.  Output is
+#pragma out written to outfile.   Outfile and infile must be distinct.
+
+"""
+repypp.py is a preprocessor for repy. It includes dependent files as needed.This is used to help the 
+programmer avoid the need to use import. They can instead use "include X" which works somewhat like "from X import *".
+
+This script tests if repypp.py check erroneous arguments(have multiple arguments).
+
```

Comment:
"If repypp works correctly, it will not accept any number of command-line arguments other than two, and will print an error message and usage string, see https://github.com/SeattleTestbed/seattlelib_v2/blob/master/repypp.py#L197-L200and https://github.com/SeattleTestbed/seattlelib_v2/blob/master/repypp.py#L41-L45"